### PR TITLE
feat: removed mongofs and added graalvm

### PR DESCRIPTION
**Pull request by stephane-segning**

### CODE REVIEW

**Review by mbunwe-victor — CHANGES_REQUESTED** [review]
Sir @stephane-segning most of the builds are not passing please check them out.

File: src/main/java/com/ssegning/lynx/lynxbackend/controller/FileApiImpl.java
Comment (nancymuyeh) [inline src/main/java/com/ssegning/lynx/lynxbackend/controller/FileApiImpl.java:1]: This implements an interface FileApi which does not exist and is causing the build to fail also making the workflow to fail . So you should do something about it

**Review by nancymuyeh — CHANGES_REQUESTED** [review]
Well done adding graalvm to the project, seems good

**Comment by stephane-segning** [comment]
> Sir @stephane-segning most of the builds are not passing please check them out.

Done!

File: src/main/java/com/ssegning/lynx/lynxbackend/controller/FileApiImpl.java
Comment (stephane-segning) [reply src/main/java/com/ssegning/lynx/lynxbackend/controller/FileApiImpl.java:1]: This was because of the frontend not well being loaded. Fixed now

**Pull request merged**